README.md

```diff
@@ -84,6 +73,6 @@ $ reshowcase start --entry=path/to/Demo.js
 $ reshowcase build --entry=path/to/Demo.js --output=path/to/bundle
 ```
 
-If you need custom webpack options, create the `.reshowcase/config.js` and export the webpack config, plugins and modules will be merged.
+If you need custom esbuild options, create the `.reshowcase/config.js` and export the esbuild config. Plugins and modules will be merged.
```

Comment:
I wonder if we should mention here that if users use their own template, they have to add the `new EventSource('/esbuild')...` script manually if they want to keep livereload working.